tests/starknet/coordinator.rs

```diff
@@ -0,0 +1,82 @@
+use std::fs;
+
+use anyhow::Error;
+use chrono;
+
+#[allow(dead_code)]
+pub enum TestMode {
+    Katana,
+    Sepolia,
+}
+
+#[allow(dead_code)]
+pub struct Coordinator {
+    id: String,
+    mode: TestMode,
+    source: String,
+    target: String,
+}
+
+#[allow(dead_code)]
+impl Coordinator {
+    pub fn new(mode: TestMode) -> Self {
+        let now = chrono::offset::Local::now();
+        let id = now.format("%Y%m%y%H%M%S").to_string();
+        let target = "./target/account-".to_string() + &id;
+        Self {
+            id,
+            mode,
+            source: "./tests/starknet/contract/account".to_string(),
+            target,
+        }
+    }
+
+    pub fn copy_template_to_target(&self) -> Result<(), Error> {
```

Comment:
The problem with such design is that `Coordinator` instance is created in the invalid state, and is unusable until methods are called in specific order. Why not extract them as a functions and use in `Coordinator`? Then whole `Coordinator` can be substituted with a simple function (idk, maybe along the lines of `prepare_account`) which is impossible to misuse.